Tools/TestWebKitAPI/Configurations/Base.xcconfig

```diff
@@ -35,6 +35,7 @@ PROJECT_HEADER_SEARCH_PATHS = $(SRCROOT)/../../Source/WebKit/Platform/cocoa $(PR
 PROJECT_HEADER_SEARCH_PATHS_cocoatouch = $(inherited) $(SRCROOT)/../../Source/WebKit/Platform/spi/ios $(SRCROOT)/../../Source/WebKit/UIProcess/ios;
 
 HEADER_SEARCH_PATHS = $(ALTERNATE_HEADER_SEARCH_PATHS) ${BUILT_PRODUCTS_DIR}/usr/local/include $(WEBCORE_PRIVATE_HEADERS_DIR)/ForwardingHeaders $(BUILT_PRODUCTS_DIR)/WebCoreTestSupport ${SRCROOT} $(PROJECT_HEADER_SEARCH_PATHS);
+SYSTEM_HEADER_SEARCH_PATHS = $(WK_PRIVATE_SDK_DIR)$(WK_ALTERNATE_WEBKIT_SDK_PATH)$(WK_LIBRARY_HEADERS_FOLDER_PATH);
```

Comment:
Without `$inherited`, are we potentially losing paths inherited from outside xcconfigs (such as command line options, or compiler built-in)? Also, unsure why this was unnecessary before, but is necessary now.

Reply:
> Without `$inherited`, are we potentially losing paths inherited from outside xcconfigs (such as command line options, or compiler built-in)?

I don't think so (iirc, built-in paths like `/usr/include` are baked into clang, not the build system. And command line options would override these). But it doesn't hurt to inherit.

> Also, unsure why this was unnecessary before, but is necessary now.

Currently, the one system header we provide in sparse SDKs is `AvailabilityProhibitedInternal.h`, which we use to disable available macros. Previously, public SDK builds had that header injected into the base SDK, where it could be found via built-in search paths.